src/pystage/core/_sensing.py

```diff
@@ -190,7 +190,13 @@ def __init__(self):
         super().__init__()
 
     def sensing_touchingobject_pointer(self):
-        pass
+        # if mouse pointer is touching sprite
+        mx, my = pygame.mouse.get_pos()
+        if self.rect.collidepoint(mx, my):
+            #print("Mouse is touching the sprite!")
+            offset = (self.rect.left - self.rect.left, self.rect.top - self.rect.top)
```

Comment:
I believe this is always (0,0)

Reply:
I tested the ball sprite in Scratch and this is very close to correct. In Scratch only if the top of the mouse pointer is touching the sprite then the block is triggered not the entire pointer. My code assumes if any part of the mouse pointer is touching the object then the block will trigger. Other then that the block worked the same as it did with a rectangular sprite.